addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5516,9 +5516,8 @@ registry.ReplaceMedia = SnippetOptionWidget.extend({
      * @see this.selectClass for parameters
      */
     async replaceMedia() {
-        // TODO for now, this simulates a double click on the media,
-        // to be refactored when the new editor is merged
-        this.$target.dblclick();
```

Comment:
Removing the double click will make `range.startContainer` to be the "replace media" button instead of the image in `openMediaDialog`, which is not really desirable:
```js
openMediaDialog(params = {}) {
        const sel = this.odooEditor.document.getSelection();
        (...)
        const $editable = $(OdooEditorLib.closestElement(range.startContainer, '.o_editable') || this.odooEditor.editable);
```

If you pass the node as parameter, you can correct this line to something like
```javascript
const $editable = $(OdooEditorLib.closestElement(params.node || range.startContainer, '.o_editable') || this.odooEditor.editable);
```

Reply:
(the res_model and res_id info will still be wrong anyway and have nothing to do with this task, there's a task for that: https://www.odoo.com/web#id=3502797&cids=1&menu_id=4720&action=333&active_id=1695&model=project.task&view_type=form

Reply:
@cammarosano Thanks for suggestions. :blush: